Jellyfin.Api/Controllers/ArtistsController.cs

```diff
@@ -86,7 +86,7 @@ public class ArtistsController : BaseJellyfinApiController
             [FromQuery] int? limit,
             [FromQuery] string? searchTerm,
             [FromQuery] string? parentId,
-            [FromQuery] string? fields,
+            [FromQuery] ItemFields[] fields,
```

Comment:
I'm slightly annoyed by the fact the type is called `ItemFields` instead of `ItemField` but I guess that's not in the scope of this PR.